Gaijinizer patch/Map067.txt

```diff
@@ -267,6 +274,7 @@ Now who was the one who said something like that?#Aesthetic
 ああああああああああああああああああああ！
 > CONTEXT: Map067/events/1/pages/1/95/Dialogue/kazuya-3
 Uwaaaaaaaaaaaaaaaa!!! 
+
 Aaaaaaaaaaaaaaaaaa!#Aesthetic
 > END STRING
```

Comment:
We might want to change this line to be the feral screaming we've done elsewhere.